src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -293,4 +295,41 @@ object Helper {
     }
     file
   }
+
+  def checkFilesExist(paths: Path*): Unit = {
+    paths.foreach(path => {
+      val file = new File(path.toString)
+      if (!file.exists()) {
+        throw new RuntimeException(s"File does not exist: ${file.getPath}")
+      }
+    })
+  }
+
+  def readArchiveFile(filePath: Path): List[String] = {
+    checkFilesExist(filePath)
```

Comment:
Is there a need for a separate check? This introduces a race condition where somebody could delete the file in between the check and the access.